src/Tinc/Package.hs

```diff
@@ -18,7 +19,13 @@ parsePackage s = case break (== '-') (reverse s) of
   _ -> Package s ""
 
 parseInstallPlan :: String -> [Package]
-parseInstallPlan = map parsePackage . concatMap (take 1 . words) . drop 2 . lines
+parseInstallPlan (lines -> ("Resolving dependencies..."
+                            : what
+                            : maybePackages))
```

Comment:
I avoid to align code in this way.  If we later rename `parseInstallPlan`, everything needs to be realigned.  In this specific case I also think that pattern guards are a valid option (see https://github.com/zalora/tinc/commit/86171ed6a49f4bbcc2c6cc0f76606be4396884e1).